paragon/core.scss

```diff
@@ -0,0 +1,4 @@
+// this file creates the core.css import here all the common styles for your theme. 
+
+// @import "./build/core/variables";
```

Comment:
should we recommend/demonstrate using `@use` vs. `@import` in the SCSS files, given `@import` is technically deprecated within `sass`?

Some related reading:
* https://sass-lang.com/documentation/at-rules/use/
* https://medium.com/@philip.mutua/difference-between-use-and-import-in-scss-1cb6f501e649